pkg/utils/read.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+func ReadBytesFromFile(filePath string) ([]byte, error) {
```

Comment:
It would be a good idea to limit how much file we are willing to read into memory